### ISSUE TITLE
German words

### ISSUE THREAD

**Issue by DiplEng** [hermitdave/FrequencyWords#14]
For german words it would be really beneficially if they could be written properly -> Nouns are written capitalized. 
So not "freund" but "Freund".

This would allow this list to be used for spellchecking.

**Comment by amfolio** [comment]
True, and also in German the pronouns "sie" and "Sie" have different meanings, so differentiating them in statistics could lead to much better results.

**Comment by BrendanMartin** [comment]
I'm also hoping this can be fixed

**Comment by hermitdave** [comment]
Let me see if I can hack this bit in @felix-schneider - might have to do a find and replace. The issue with nouns @DiplEng and @amadeomano is knowing how to identify them programmatically. The data is just a sentence - most likely manually created
